extensions/apiconnect/README.md

```diff
@@ -0,0 +1,64 @@
+# @loopback/apiconnect
+
+This module extends LoopBack with ability to integrate with
+[IBM API Connect](https://www.ibm.com/cloud/api-connect).
+
+## Stability: ⚠️Experimental⚠️
+
+> Experimental packages provide early access to advanced or experimental
+> functionality to get community feedback. Such modules are published to npm
+> using `0.x.y` versions. Their APIs and functionality may be subject to
+> breaking changes in future releases.
+
+## Installation
+
+```sh
+npm i @loopback/apiconnect --save
+```
+
+## Usage
+
+The component should be loaded in the constructor of your custom Application
+class.
+
+Start by importing the component class:
+
+```ts
+import {ApiConnectComponent} from '@loopback/apiconnect';
+```
+
+In the constructor, add the component to your application:
+
+```ts
+this.component(ApiConnectComponent);
+```
+
+The component requires a configuration for API Connect extension for OpenAPI
+spec:
+
+```ts
+const apiConnectOptions: ApiConnectSpecOptions = {
+  targetUrl: 'http://localhost:3000/test-service',
```

Comment:
Question: if I want to use the same url for the APIC-flavored OpenAPI spec, can I not specify the `targetUrl`?

Reply:
The `targetUrl` is required as it tells API Connect gateway where the REST APIs are hosted. We cannot infer it.